Library/Aliases/git-hub

```diff
@@ -0,0 +1 @@
+../Formula/git-cmd-hub.rb
```

Comment:
I think this comes back to the naming issue discussed on the original pull request. The software is actually called `git-hub` (see the [author's response](https://github.com/Homebrew/homebrew/pull/31886#issuecomment-54654166) for reasoning), and having the correct name available via `brew search` seems important for discoverability.